endaq/batch/core.py

```diff
@@ -247,6 +247,13 @@ def __init__(
         :param accel_end_margin: the numper of samples after which to reject
             recording data; cannot be used in conjunction with
             `accel_end_time`
+        :param accel_integral_tukey_percent: the alpha parameter of a tukey
```

Comment:
nbd, but Tukey should be capitalized in the docstring (where it isn't referring to a variable/parameter/function name).

Reply:
fixed in b01ea5c96503ca46794748bf9b4dba3a5ec9827c. thanks!